mypyc/genops.py

```diff
@@ -4967,6 +4979,23 @@ def add_throw_to_generator_class(self,
         fn_info.generator_class.ir.methods['throw'] = throw_fn_ir
         self.functions.append(throw_fn_ir)
 
+    def add_close_to_generator_class(self, fn_info: FuncInfo) -> None:
+        """Generates the '__close__' method for a generator class."""
+
+        # TODO: Currently this is a dummy method does the exact same thing as
+        # __iter__, we need to actually fill this in.
+        self.enter(fn_info)
+        self_target = self.add_self_to_env(fn_info.generator_class.ir)
+        self.add(Return(self.read(self_target, fn_info.fitem.line)))
```

Comment:
Have it raise an exception with an unimplemented message or something instead.

Reply:
Done!